### DISCUSSION TITLE
[Feature Request] Disable/Customize shortcuts

### DISCUSSION THREAD

**Discussion by rozetta1125** [arnoudkooi/SN-Utils#436] [Ideas]
Currently, The shortcut for Insert and Stay is close to Window Screenshot shortcut, thus, we encounter a problem where a user accidentally inserted a new record. Upon investigation, this can also occur with lots of records (Notification, Business Rule, etc) which can pose a problem.

Our Team uses shortcuts a lot, so it would be nice if we could modify, or disable some shortcuts.
### Suggestion
- Shortcuts are disabled by default.
- Ability to customize the shortcuts.
- Disable some shortcuts.

Best Regards
Thank you.

**Comment by Dolphinbi123** [comment]
Oh. I am also eager for an updated feature to avoid the same issue above.

**Comment by toshinorioike** [comment]
I have discussed the dangers related to this subject in my office.
Please add the option.

**Comment by nnghiaYugi** [comment]
I also have the same concern, it is great if we can change that shortcut.

**Comment by arnoudkooi** [comment]
In the longer term, I may see if I can fully customize the shortcuts.
In the shorter term, I'll add a general setting that disables all shortcuts.

I'm moving this to a discussion and will comment when there is an update.
